FirebaseRemoteConfig/Sources/RCNConfigExperiment.m

```diff
@@ -32,7 +32,9 @@ @interface RCNConfigExperiment ()
     NSMutableArray<NSData *> *experimentPayloads;  ///< Experiment payloads.
 @property(nonatomic, strong)
     NSMutableDictionary<NSString *, id> *experimentMetadata;  ///< Experiment metadata
-@property(nonatomic, strong) RCNConfigDBManager *DBManager;   ///< Database Manager.
+@property(nonatomic, strong)
+    NSMutableArray<NSData *> *activatedExperimentPayloads;   ///< Activated experiment payloads.
```

Comment:
Nit: It'd be helpful to use either "active" and "activated" consistently

Reply:
And does this need to be initialized in `initWithDBManager:` similar to `_experimentPayloads = [[NSMutableArray alloc] init];`?

Reply:
Switched to only "active". And I think `loadExperimentsFromTable` will always init the experiments variables even if it doesn't have that key in the table. But I added the explicit init to be in line with the other vars